.expeditor/config.yml

```diff
@@ -110,19 +110,19 @@ subscriptions:
   # the omnibus/docker/gem chain
   - workload: artifact_published:unstable:chef:{{version_constraint}}
```

Comment:
we might also have to rework this as well, since we arnt doing any artifactory publish currently, with habitat anymore.

Reply:
I've moved the docker/build pipeline to buildkite_hab_build_group_published workload, guessing that will be triggered when the hab images are published to the builder and we need to create the docker images with the new ones. @sean-simmons-progress what do you think?